scripts/jenkins/jenkinsfiles/Jenkinsfile-Nightly

```diff
@@ -1,6 +1,6 @@
 @Library('test-shared-library') _
 
-def DEFAULT_NODE_LABEL = 'h2o-3 && docker && !mr-0xc8 && (!micro || micro_21)'
+def DEFAULT_NODE_LABEL = 'DC && linux && docker'
```

Comment:
Shouldn't the `h2o-3` label be there? I remember Devops team complaining that we're using to many nodes and other projects starve

Reply:
Btw, what does `DC` mean?

Reply:
@DinukaH2O